src/resources/common.yml

```diff
@@ -50,8 +50,8 @@ language:
   en: Language
   de: Sprache
 invalidSlug:
-  en: The URL path must contain only lower case letters, hyphons, slashes and digits
-  de: Der URL-Pfad darf nur Kleinbuchstaben, Bindestriche, Schrägstriche und Ziffern enthalten
+  en: Normally the URL path must contain only lower case letters, hyphons, slashes and digits
+  de: Normalerweise der URL-Pfad darf nur Kleinbuchstaben, Bindestriche, Schrägstriche und Ziffern enthalten
```

Comment:
"Der URL-Pfad sollte nur Kleinbuchstaben, Bindestriche, Schrägstriche und Ziffern enthalten"

"The URL path should contain only lower case letters, hyphens, slashes and digits"

Reply:
Done